bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -207,6 +207,12 @@ spec:
           verbs:
```

Comment:
Is this commit also part of the fix? If so, please add `fixes bz#____` to the commit message.

Reply:
Actually, this commit does not `fixes` the bug, It's a prep work for https://github.com/openshift/windows-machine-config-operator/pull/645/commits/bf2a331e4d0043ffbfa422f233de741d15e5a119.

Thanks for calling this out, will add more context in the commit message.